MapboxCoreNavigation/RouteController.swift

```diff
@@ -216,6 +216,11 @@ extension RouteController: CLLocationManagerDelegate {
         lastUserDistanceToStartOfRoute = Double.infinity
     }
     
+    /**
+     Given a users current location, returns a `bool` whether they are currently on the route.
```

Comment:
Use “Boolean” instead of `bool`, because it’s spelled `Bool` and Swift but `BOOL` in Objective-C.